style.css

```diff
@@ -231,14 +232,14 @@
   }
   
   /* static animation for signal failure effect */
-  /*
+  
   .tv-static {
     position: fixed;
     z-index: -1;
     margin: auto;
     width: 100vw;
     height: 100vh;
-    background-image: repeating-conic-gradient(#EC008E, #7cccfc 0.00085%);
+    background-image: repeating-conic-gradient(var(--sharp-retroPink), var(--skyBlue-color) 0.00085%);
```

Comment:
Really interesting to see how you put this together!

Reply:
Thank You, it was great using the CSS Variables for the colours 🥳